pkg/runctl/controller.go

```diff
@@ -36,6 +36,9 @@ const kind = "PipelineRuns"
 var heartbeatIntervalSeconds int64 = 60
 var heartbeatTimer int64 = 0
 
+// Used for metering
+var meteringHistogramInterval = time.Second * 3
```

Comment:
That will be lot's of observations. Do you know what the scrape interval is?

Reply:
I could not see that we overwrite the default value of 1 minute (https://prometheus.io/docs/prometheus/latest/configuration/configuration/).
So I set it to 1 minute